### PR TITLE
Modified to use the frame and exposure keywords added by marci2isis i…

### PR DESCRIPTION
**Pull request by scsides**
Modified to use the frame and exposure keywords added by marci2isis instead of the varexp.tab file

## Description

Took out the code for reading the varexp.tab file and used the instrument keywords instead

## Related Issue

#2034

## Motivation and Context

marci2isis now read the file and addes the keywords

## Types of changes

- [] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/mro/apps/marcical/marcical.xml [hunk 1]
```diff
@@ -44,6 +44,10 @@ K.D. Supulver, and P.C. Thomas
     <change name="Moses Milazzo" date="2014-12-08">
       Fixed a bug in exposure time accounting. Fixes #1860.
     </change>
+    <change name="Stuart Sides" date="2019-11-08">
+      Modified to use the exposure duration from the new label keywords added by
+      marci2isis.
```
Comment (jessemapel) [inline isis/src/mro/apps/marcical/marcical.xml]: Add the github issue number

File: isis/src/mro/apps/marcical/main.cpp [hunk 2]
```diff
@@ -364,56 +364,38 @@ void IsisMain() {
...
+  // the array when converting them from the keywords. This calibration code should be 
+  // able to be simplified significantly by not removing the frameNumber=0 exposure/frame and
+  // removing the first frame ifs in the calibration code below.
+  PvlKeyword expTimesKey = inst["VariableExposureDuration"];
+  PvlKeyword frameNumbersKey = inst["FrameNumber"];
+  for (int i=0; i<expTimesKey.size(); i++) {
+    if (toInt(frameNumbersKey[i]) != 0) {
+      exptime.push_back(toDouble(expTimesKey[i]));
+      frameseq.push_back(toInt(frameNumbersKey[i]));
+    }
```
Comment (jessemapel) [inline isis/src/mro/apps/marcical/main.cpp:392]: I think you can remove this by just removing the else part of the if/else statements later on lines 438 and 461.

**Review by jessemapel — COMMENTED** [review]
Looks reasonable enough based on ingestion changes

**Review by krlberry — DISMISSED** [review]
:+1: This looks good to me (after the comments that Jesse made.) Especially appreciate the added comments!

**Review by ladoramkershner — APPROVED** [review]

**Pull request merged**